source/JavaPropertiesParser.Tests/ParserFixture.cs

```diff
@@ -350,37 +351,49 @@ public void CanParseAValueWithALogicalNewLine()
         }
 
         [Test]
-        public void CanParseAValueWithAPhysicalNewLine()
+        [TestCase("value-with-physical-crlf-newline.properties")]

```

Comment:
Original test file is duplicated and updated where 1 file with CRLF (Windows) and the other with LF (Linux / MacOS)